tests/test.js

```diff
@@ -3,28 +3,42 @@ import { readdirSync, readFileSync } from "node:fs";
 import path from "node:path";
 import { JSDOM } from "jsdom";
 import { fileURLToPath } from "url";
+import { execa } from "execa";
 
 const __dirname = fileURLToPath(new URL(".", import.meta.url));
 
 describe("Embroider build", function () {
   let indexHtml;
 
-  const scenarios = readdirSync(path.join(__dirname, "..", "test-apps"));
+  const scenarios = readdirSync(path.join(__dirname, "..", "test-apps")).filter(
+    (file) =>
+      [
+        // skip these test apps
+        "external-resource-missing-integrity-hash",
+        "external-resource-with-integrity-hash",
+      ].includes(file) === false,
+  );
 
-  for (const scneario of scenarios) {
-    describe(`Scenario: ${scneario}`, function () {
-      before(function () {
+  for (const scenario of scenarios) {
+    describe(`Scenario: ${scenario}`, function () {
+      before(async function () {
+        this.timeout(60000);
         try {
+          const appPath = path.join("..", "test-apps", scenario);
+          await execa("pnpm", ["build"], {
```

Comment:
I can remove this if you'd prefer to build these scenarios manaully, but we _do_ need to be able to run the builds for the other tests.

Reply:
Oh. I feel I misunderstood. I assumed you wanted to generate the test apps as part of the test. In the sense of running `ember new` in the tests. Instead of having the apps committed as fixtures to the repo.

Building the test apps as part of the test is fully fine. And may even reduce barrier for contributions. With current serup it's easy to forget (re-)building the test app. And being confused about the results.

I'm sorry for the confusion!

Reply:
No problem 😄 

I've added this back and removed the build script and step from the workflow.